config/system/default.yaml

```diff
@@ -44,3 +44,4 @@ topology_hook: bin/dump_network
 # usi url for DAQ to connect to
 usi_setup: 
   url: localhost:5000
+  grpc_timeout_sec: 10
```

Comment:
just rpc_timeout_sec -- there's no need to bleed the implementation of "this is grpc"...

Reply:
renamed. PTAL